splice_machine/src/main/java/com/splicemachine/client/SpliceClient.java

```diff
@@ -154,7 +149,19 @@ public static DataSource getConnectionPool(boolean debugConnections, int maxConn
                     pool = new ComboPooledDataSource();
                     try {
                         pool.setDriverClass("com.splicemachine.db.jdbc.Driver40");
+
+                        // parse and mask password in JDBC url in order to make it not logged
+                        String password = parseJDBCPassword(connectionString);
+                        if (password != null) {
+                            connectionString = connectionString.replace(
+                                    "password="+password+";", "");
+                            connectionString = connectionString.replace(
```

Comment:
Duplicate line

Reply:
This will first remove if the URL is like `....password=XXXX;username=XXX`, which have semicolon after password, and then remove the case like "username=XXXX;password=", which don't have anything after password.

Reply:
I see, thanks!